back/engines/commercial/analysis/app/lib/analysis/auto_tagging_method/nlp_topic.rb

```diff
@@ -41,7 +41,7 @@ def run_topic_modeling_prompt(project_title, inputs)
 
     def inputs_prompt(inputs, project_title)
       inputs_text = input_to_text.format_all(inputs)
-      LLM::Prompt.new.fetch('topic_modeling', project_title: project_title, inputs_text: inputs_text, max_topics: max_topics(inputs.size), language: Locale.monolingual&.language_copy)
```

Comment:
So, this was a bug? Were we defaulting to Spanish when this returned `nil` (on multi-lingual platforms)?